test/pytest/test_bridge.py

```diff
@@ -76,97 +94,156 @@ async def assert_msg(self, expected_channel, **kwargs) -> None:
         msg = await self.next_msg(expected_channel)
         assert msg == msg | {k.replace('_', '-'): v for k, v in kwargs.items()}
 
+    # D-Bus helpers
+    internal_bus: str = ''
+
+    async def open_internal_bus(self):
```

Comment:
Calling this `get_internal_bus` might be better. I was confused by the `if bus is None: bus = open_internal_bus()` pattern below, and thought it would open a new channel every time, which felt pretty useless.

Reply:
Well ,it opens it if it's not already.  Sure, makes sense, though.